From 3f16a6024bcf95f63c80340b5d518727af93d946 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@hadrons.org>
Date: Wed, 3 Nov 2021 00:20:50 +0100
Subject: [PATCH libaio 07/26] Move semicolon to SYMVER and DEFSYMVER call
 sites

These look like function macros, so let's unify their call sites to use
function syntax with a final semicolon.

Origin: vendor
Forwarded: https://marc.info/?l=linux-aio&m=164999309420541
Signed-off-by: Guillem Jover <guillem@hadrons.org>
---
 src/io_cancel.c     | 2 +-
 src/io_getevents.c  | 2 +-
 src/io_queue_wait.c | 2 +-
 src/syscall.h       | 4 ++--
 4 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/io_cancel.c b/src/io_cancel.c
index 2f0f5f4..441806d 100644
--- a/src/io_cancel.c
+++ b/src/io_cancel.c
@@ -20,4 +20,4 @@
 #include "syscall.h"
 
 io_syscall3(int, io_cancel_0_4, io_cancel, io_context_t, ctx, struct iocb *, iocb, struct io_event *, event)
-DEFSYMVER(io_cancel_0_4, io_cancel, 0.4)
+DEFSYMVER(io_cancel_0_4, io_cancel, 0.4);
diff --git a/src/io_getevents.c b/src/io_getevents.c
index 90d6081..88d285a 100644
--- a/src/io_getevents.c
+++ b/src/io_getevents.c
@@ -32,4 +32,4 @@ int io_getevents_0_4(io_context_t ctx, long min_nr, long nr, struct io_event * e
 	return __io_getevents_0_4(ctx, min_nr, nr, events, timeout);
 }
 
-DEFSYMVER(io_getevents_0_4, io_getevents, 0.4)
+DEFSYMVER(io_getevents_0_4, io_getevents, 0.4);
diff --git a/src/io_queue_wait.c b/src/io_queue_wait.c
index 538d2f3..6f69a51 100644
--- a/src/io_queue_wait.c
+++ b/src/io_queue_wait.c
@@ -28,4 +28,4 @@ int io_queue_wait_0_4(io_context_t ctx, struct timespec *timeout)
 {
 	return io_getevents(ctx, 0, 0, NULL, timeout);
 }
-DEFSYMVER(io_queue_wait_0_4, io_queue_wait, 0.4)
+DEFSYMVER(io_queue_wait_0_4, io_queue_wait, 0.4);
diff --git a/src/syscall.h b/src/syscall.h
index d2a117b..f7ce979 100644
--- a/src/syscall.h
+++ b/src/syscall.h
@@ -6,10 +6,10 @@
 #define SYMSTR(str)	_SYMSTR(str)
 
 #define SYMVER(compat_sym, orig_sym, ver_sym)	\
-	__asm__(".symver " SYMSTR(compat_sym) "," SYMSTR(orig_sym) "@LIBAIO_" SYMSTR(ver_sym));
+	__asm__(".symver " SYMSTR(compat_sym) "," SYMSTR(orig_sym) "@LIBAIO_" SYMSTR(ver_sym))
 
 #define DEFSYMVER(compat_sym, orig_sym, ver_sym)	\
-	__asm__(".symver " SYMSTR(compat_sym) "," SYMSTR(orig_sym) "@@LIBAIO_" SYMSTR(ver_sym));
+	__asm__(".symver " SYMSTR(compat_sym) "," SYMSTR(orig_sym) "@@LIBAIO_" SYMSTR(ver_sym))
 
 #if defined(__i386__)
 #include "syscall-i386.h"
-- 
2.43.0

